pkg/channel/consolidated/reconciler/controller/controller.go

```diff
@@ -89,11 +89,8 @@ func NewController(
 		logger.Panicf("unable to process Kafka channel's required environment variables: %v", err)
 	}
 
-	if env.Image == "" {
-		logger.Panic("unable to process Kafka channel's required environment variables (missing DISPATCHER_IMAGE)")
-	}
-
 	r.dispatcherImage = env.Image
+	r.serviceAccount = env.ServiceAccount
```

Comment:
maybe it's good to have `dispatcher` as part of the "serviceAccount" var name here and below ? or an acronym that shows this is a "dispatcher" service account similar to "dispatcherImage" ?

Reply:
+1